### ISSUE TITLE
Group Policy Settings

### ISSUE THREAD

**Issue by jeroenebus** [MicrosoftDocs/windowsserverdocs#2043]
In the Group Policy Settings section you are advising:

Click Recovery, and then double-click Allow restore of system to default state. Select the radio button for Enabled, and then click the OK button to save.

Is this a typo? I'm thinking that it is not likely for a VDI user to reset the VDI in the Default Configuration state, isn't it?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a83e1751-011c-9add-2c0a-63159fef97da
* Version Independent ID: 119b5192-3775-f71e-579d-9ead105ed1b0
* Content: [Recommended configuration for VDI desktops](https://docs.microsoft.com/en-us/windows-server/remote/remote-desktop-services/rds-vdi-recommendations)
* Content Source: [WindowsServerDocs/remote/remote-desktop-services/rds-vdi-recommendations.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/remote/remote-desktop-services/rds-vdi-recommendations.md)
* Product: **windows-server-threshold**
* GitHub Login: @jaimeo
* Microsoft Alias: **elizapo**

**Comment by jaimeo** [comment]
Good catch; that is a typo. It should be "disable." We're about to publish an update of this topic and will be sure that gets in there.

**Comment by IngridAtMicrosoft** [comment]
#please-close
